From 0f543554e07b22895f523b236364a509a71bce1d Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <ofourdan@redhat.com>
Date: Thu, 15 Dec 2022 16:57:01 +0100
Subject: [PATCH 036/105] modesetting: Document the "Atomic" option

The modesetting driver has atomic modesetting disabled by default but
can be enabled (if supported) using a configuration option.

Add this option in the man page.

Signed-off-by: Olivier Fourdan <ofourdan@redhat.com>
Reviewed-by: Martin Roukala <martin.roukala@mupuf.org>
---
 hw/xfree86/drivers/modesetting/modesetting.man | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hw/xfree86/drivers/modesetting/modesetting.man b/hw/xfree86/drivers/modesetting/modesetting.man
index 9e40b04f3..485beff48 100644
--- a/hw/xfree86/drivers/modesetting/modesetting.man
+++ b/hw/xfree86/drivers/modesetting/modesetting.man
@@ -120,6 +120,10 @@ applications will still have the discretion to not use tearing prevention.
 The default is
 .B off.
 .TP
+.BI "Option \*qAtomic\*q \*q" boolean \*q
+Enable atomic modesetting when supported.  The default is
+.B off.
+.TP
 .SH "SEE ALSO"
 @xservername@(@appmansuffix@), @xconfigfile@(@filemansuffix@), Xserver(@appmansuffix@),
 X(@miscmansuffix@)
-- 
2.45.2

